### ISSUE TITLE
Ctrl-C Interrupt Breaks `terraform` executable. 

### ISSUE THREAD

**Issue by anders-s** [warrensbox/terraform-switcher#66]
MacOS Mojave 10.14.6, `tfswitch Version: 0.7.737`

`ctrl-c` signaling while in the `tfswitch` versions menu will result in no terraform executable being linked to the system wide `terraform` command. 

```
$ terraform -v 
Terraform v0.12.20
Your version of Terraform is out of date! The latest version
is 0.12.21. You can update by downloading from https://www.terraform.io/downloads.html
$ tfswitch
Use the arrow keys to navigate: ↓ ↑ → ← 
? Select Terraform version: 
  ▸ 0.12.20 *recent
    0.11.14 *recent
    0.11.13 *recent
    0.12.21
↓   0.12.19
2020/02/27 17:48:16 Prompt failed ^C
$ terraform -v
bash: /usr/local/bin/terraform: No such file or directory
```

**Comment by warrensbox** [comment]
I see the issue. I will try to fix this by this week @anders-s

**Comment by warrensbox** [comment]
Fixed with #68 
`brew upgrade warrensbox/tap/tfswitch` to get the latest version
